org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/batch/messages.properties

```diff
@@ -353,7 +347,7 @@ misc.usage = {1} {2}\n\
 \    -noExit            do not call System.exit(n) at end of compilation (n==0\n\
 \                       if no error)\n\
 \    -repeat <n>        repeat compilation process <n> times for perf analysis\n\
-\    -inlineJSR         inline JSR bytecode (implicit if target >= 1.5)\n\
+\    -inlineJSR         inline JSR bytecode (implicit if target >= 1.8)\n\
```

Comment:
I assume we can ignore this argument (no need to report an error if specified)?
I also assume we can deprecate `CompilerOptions.OPTION_InlineJsr` now (it is "internal" constant, but let be careful here as it may be used by clients since ever)?

Reply:
> but let be careful here as it may be used by clients since ever 

Good point. In fact via org.eclipse.jdt.core.prefs the option can be used without even touching internal classes.

Option_InlineJsr is also mirrored as `JavaCore.COMPILER_CODEGEN_INLINE_JSR_BYTECODE`.

Both this option key and the CLI argument are API, so perhaps we can handle both in parallel:
* deprecate constants, internal and API, with a comment that for all supported versions the value ENABLED is implied.
* change the help here to smth like "inline JSR bytecode (legacy option, is always enabled in all supported target versions)

Reply:
I've partly already fixed that in first changes round, but now pushed updated version to deprecate  all these "inline JSR" constants/fields